cms/envs/aws.py

```diff
@@ -534,3 +531,7 @@
 
 # Allow extra middleware classes to be added to the app through configuration.
 MIDDLEWARE_CLASSES.extend(ENV_TOKENS.get('EXTRA_MIDDLEWARE_CLASSES', []))
+
+########################## Derive Any Derived Settings  #######################
+
+derive_settings(sys.modules[__name__])
```

Comment:
We could simplify this by just passing in `__name__`, and having `derive_settings()` fetch the module from `sys.modules`

Reply:
I like it - even simpler. Will do.